design/txn_author_agreement.md

```diff
@@ -91,6 +92,7 @@ Add new version of AML.
     "reqId": INT,
     "operation": {
         "type": INT,
+        "version": <str>
```

Comment:
Please describe how AML is stored in state

Reply:
ok